src/states/node.rs

```diff
@@ -1684,10 +1683,9 @@ impl Node {
                 .into_iter()
                 .filter(|target| !sent_to.contains(target))
                 .collect();
-            let new_sent_to = if self.peer_mgr
-                .routing_table()
-                .our_group_prefix()
-                .matches(routing_msg.dst.name()) {
+            // TODO: shouldn't sent_to be a BTreeSet? This will potentially duplicate self.name()
+            // in sent_to.
```

Comment:
Good point. All we use it for is `contains`. Feel free to add it to this PR, if you prefer.